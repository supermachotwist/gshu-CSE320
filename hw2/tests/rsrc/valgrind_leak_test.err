==22636== Memcheck, a memory error detector
==22636== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==22636== Using Valgrind-3.13.0 and LibVEX; rerun with -h for copyright info
==22636== Command: bin/finddup tests/rsrc/quick_test_names
==22636== 
build list...sort...==22636== Conditional jump or move depends on uninitialised value(s)
==22636==    at 0x109A32: comp1 (finddup.c:251)
==22636==    by 0x4E7DF72: msort_with_tmp.part.0 (msort.c:124)
==22636==    by 0x4E7E20D: msort_with_tmp (msort.c:117)
==22636==    by 0x4E7E20D: msort_with_tmp.part.0 (msort.c:54)
==22636==    by 0x4E7E708: msort_with_tmp (msort.c:242)
==22636==    by 0x4E7E708: qsort_r (msort.c:254)
==22636==    by 0x1098C0: finddup_main (finddup.c:213)
==22636==    by 0x1092D9: main (main.c:10)
==22636== 
==22636== Conditional jump or move depends on uninitialised value(s)
==22636==    at 0x109A32: comp1 (finddup.c:251)
==22636==    by 0x4E7DF72: msort_with_tmp.part.0 (msort.c:124)
==22636==    by 0x4E7E708: msort_with_tmp (msort.c:242)
==22636==    by 0x4E7E708: qsort_r (msort.c:254)
==22636==    by 0x1098C0: finddup_main (finddup.c:213)
==22636==    by 0x1092D9: main (main.c:10)
==22636== 
scan1...scan2...done
==22636== 
==22636== HEAP SUMMARY:
==22636==     in use at exit: 2,952 bytes in 2 blocks
==22636==   total heap usage: 14 allocs, 12 frees, 34,384 bytes allocated
==22636== 
==22636== LEAK SUMMARY:
==22636==    definitely lost: 0 bytes in 0 blocks
==22636==    indirectly lost: 0 bytes in 0 blocks
==22636==      possibly lost: 0 bytes in 0 blocks
==22636==    still reachable: 2,952 bytes in 2 blocks
==22636==         suppressed: 0 bytes in 0 blocks
==22636== Reachable blocks (those to which a pointer was found) are not shown.
==22636== To see them, rerun with: --leak-check=full --show-leak-kinds=all
==22636== 
==22636== For counts of detected and suppressed errors, rerun with: -v
==22636== Use --track-origins=yes to see where uninitialised values come from
==22636== ERROR SUMMARY: 2 errors from 2 contexts (suppressed: 0 from 0)
