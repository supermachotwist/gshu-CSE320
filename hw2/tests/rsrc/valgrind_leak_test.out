

List of files with duplicate contents (includes hard links)

FILE: tests/rsrc/test_tree/file1
DUP:  tests/rsrc/test_tree/file1.dup
